cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -94,29 +87,20 @@ func (cm *certManagerClient) Images() ([]string, error) {
 	return images, nil
 }
 
-// EnsureWebhook makes sure the cert-manager Web-hook is Available in a cluster:
+// EnsureCertManager makes sure the cert-manager Web-hook is Available in a cluster:
```

Comment:
I suggest removing the "webhook is available" part since we are no longer doing that.

```suggestion
// EnsureCertManager ensures cert-manager is installed correctly by creating some test certs in the `cert-manager-test` namespace.
```

Reply:
Done :)